app.json

```diff
@@ -0,0 +1,3 @@
+{
+  "expo": {}
+}
```

Comment:
that was most probably added because of running expo in the root of the monorepo.

Reply:
nooo